docs/glossary.md

```diff
@@ -194,14 +212,21 @@ Express
     A JavaScript HTTP server with a simple API to build custom applications.
     Volto uses it as its server.
 
-Server-Side Rendering (SSR)
-    When first loading any Plone page, users will get HTML markup that closely matches the final DOM structure of the React components used to render that page.
+SSR
+server-side rendering
+    When first loading any Plone page, SSR will send HTML markup that closely matches the final DOM structure of the React components used to render that page.
+
+    More generally, when a client sends a request to an SSR page, the HTML for the page is created on the server on each request.
+    SSR enables a developer to customize a website per request and per user.
+    In addition, SSR can improve performance and search engine optimization (SEO) for a website.
```

Comment:
I would write it like so:
```suggestion
    When an HTTP client (web browser) sends a request, the HTML markup for the page is created on the server.
    
    When loading a page in  Volto the first time, SSR will send HTML markup that closely matches the final DOM structure of the React components used to render that page. 
    After initial loading the {term}`hydration` mechanism is taking over and then rendering happens browser side.
    
    When loading a page in ClassicUI the full page is loaded on every request (rarely HTML-snippets are loaded, i.e. for overlays).

    SSR enables a developer to customize a website per request and per user.
    In addition, SSR can improve performance and search engine optimization (SEO) for a website.
```

Reply:
Also, DOM should be defined as term then.

Reply:
Rewritten in [5ace25b](https://github.com/plone/documentation/pull/1416/commits/5ace25b34ae9dd83453632a092a971d38098c701). Please let me know if this hits the mark.

Reply:
> Also, DOM should be defined as term then.

Done in [c538e9c](https://github.com/plone/documentation/pull/1416/commits/c538e9cde2e2b7001b5f433ed3d63671120a7ff2)